distributed/tests/test_as_completed.py

```diff
@@ -232,6 +232,19 @@ def test_as_completed_with_results_no_raise(client):
     assert dd[z][0] == 2
 
 
+@gen_cluster(client=True)
+async def test_str(c, s, a, b):
+    futures = c.map(inc, range(3))
+    ac = as_completed(futures)
+    assert "waiting=3" in str(ac)
+    assert "waiting=3" in repr(ac)
+    assert "done=0" in str(ac)
+    assert "done=0" in repr(ac)
+
+    await ac.__anext__()
+    assert "done=2" in str(ac)
```

Comment:
Locally I'm getting occasional test failures from this assert statement

```
E       AssertionError: assert 'done=2' in '<as_completed: waiting=2 done=0>'
E        +  where '<as_completed: waiting=2 done=0>' = str(<as_completed: waiting=2 done=0>)
```

Why should we expect two futures to be done as opposed to one (for example)?